docker/alpine/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.12-alpine3.10 as builder
+
+RUN apk add --no-cache git gcc g++ ca-certificates \
+  && go get -u github.com/drakkan/sftpgo
+
+FROM alpine:3.10
+
+RUN  apk add --no-cache ca-certificates su-exec \
+  && mkdir -p /data /etc/sftpgo
+
+COPY --from=builder /go/bin/sftpgo /bin/
+COPY docker-entrypoint.sh /bin/entrypoint.sh
+RUN chmod +x /bin/entrypoint.sh
+
+VOLUME /data
```

Comment:
probably this should include the config dir too

```
VOLUME ["/data", "/path/to/config_dir"]
```

Reply:
My first DockerFile included the direcotry of config but, config directory is not required as a volume because we can configure sftpgo with en file.
I don't know if it's a good plan to include and persist default configuration in image if someone want configure product with env values